x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_creation/basic_license_essentials_tier/create_rules_bulk.ts

```diff
@@ -8,38 +8,47 @@
 import expect from '@kbn/expect';
 
 import { DETECTION_ENGINE_RULES_BULK_CREATE } from '@kbn/security-solution-plugin/common/constants';
-import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { EsArchivePathBuilder } from '../../../../../es_archive_path_builder';
+import { FtrProviderContext } from '../../../../../ftr_provider_context';
 import {
-  createSignalsIndex,
-  deleteAllRules,
-  deleteAllAlerts,
   getSimpleRule,
   getSimpleRuleOutput,
   getSimpleRuleOutputWithoutRuleId,
   getSimpleRuleWithoutRuleId,
   removeServerGeneratedProperties,
   removeServerGeneratedPropertiesIncludingRuleId,
-} from '../../utils';
+  updateUsername,
+} from '../../../utils';
+import {
+  createAlertsIndex,
+  deleteAllRules,
+  deleteAllAlerts,
+} from '../../../../../../common/utils/security_solution';
 
-// eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext): void => {
   const esArchiver = getService('esArchiver');
   const supertest = getService('supertest');
   const log = getService('log');
   const es = getService('es');
-
-  describe('create_rules_bulk', () => {
+  // TODO: add a new service for loading archiver files similar to "getService('es')"
```

Comment:
I agree with [georgii](https://github.com/elastic/kibana/pull/172173#discussion_r1424413677) that we probably don't need to keep adding this comment, but I appreciate the commitment!